packages/svelte/src/compiler/phases/2-analyze/validation.js

```diff
@@ -362,6 +362,21 @@ function validate_block_not_empty(node, context) {
  * @type {Visitors}
  */
 const validation = {
+	ExpressionStatement(node, { state }) {
+		if (node.expression.type === 'NewExpression' && node.expression.callee.type === 'Identifier') {
+			const binding = state.scope.get(node.expression.callee.name);
+			if (
+				binding?.kind === 'normal' &&
+				binding.declaration_kind === 'import' &&
+				// Theoretically someone could import a class from a `.svelte.js` module, but that's too rare to worry about
```

Comment:
I feel like this will become very common since class are kinda the blessed way...are we sure we want to add this warn?

Reply:
Only warning on default exports could help mitigate this a bit btw